.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/CVS": true,
+    "**/.DS_Store": true
+  },
+  "search.exclude": {
+    "**/dist": true,
+    "**/bin": true
+  },
```

Comment:
My guess is these were generated from sf plugin generator and we're being a good citizen to follow the practice? I guess what I'm trying to determine is if we need settings.json at all?

Reply:
Yes they were generated by the sf plugin generator.... and yes I think we should keep settings.json as it has useful settings for vscode (specially the formatOnSave, where it would auto-format as soon as you save a file)